R/class-workbook.R

```diff
@@ -6782,7 +6782,7 @@ wbWorkbook <- R6::R6Class(
 
       sheet <- as.character(sheet)
       if (has_illegal_chars(sheet)) {
-        stop("illegal characters found in sheet. Please remove. See ?openxlsx::clean_worksheet_name")
+        stop("illegal characters found in sheet. Please remove. See ?openxlsx2::clean_worksheet_name")
```

Comment:
imho we should simply apply this here and notify warning or probably simply message the user. after all what do we gain from the error? probably an annoyed user we guide to some help page explaining open XML.

Reply:
Yes possibly, do you want me to open an issue on this topic?

Reply:
Yes, agree. It was just a suggestion for the future